docs/src/inputs/prompt/devSandbox/devSandbox.jsx

```diff
@@ -9,7 +9,7 @@ import Highlighter from '../../../global/highlighter';
 import Main from '../../../global/main';
 import MarkdownContainer from '../../../global/markdownContainer';
 import ExampleDropdownButtonPrompt from '../examples/exampleDropdownButtonPrompt';
```

Comment:
What about this one?  This path is correct?

Reply:
That seems to be correct.

Reply:
nm, it the file seems to be in the wrong place.

Reply:
Fixed.

Reply:
Looks good.  Resolving.